### DISCUSSION TITLE
Saving pipelineruns log

### DISCUSSION THREAD

**Discussion by hyyyyjin** [tektoncd/pipeline#7115] [Q&A]
I am using Task, Pipeline, Pipelineruns 

Define each task - Git clone, build, image push 
Define pipeline - call above all tasks (step by step)
Define pipelineurns - trigger pipeline (pipelineref) 

What i want to try is to save each pipelineruns step's log in PVC or Object Storage.
(for example, when i call pipelineurns as named **pr-1, pr-2, pr-3**)

Log should be distinguish each other because of other service fetch those logs'
Is it possible?

**Comment by jimmyjones2** [comment]
https://tekton.dev/docs/results/
